tests/suites/test_suite_ecp.function

```diff
@@ -1602,6 +1605,20 @@ void ecp_mod_read_write(char *input_A, int id, int ctype)
     TEST_EQUAL(limbs, rX.limbs);
     ASSERT_COMPARE(rA.p, bytes, rX.p, bytes);
 
+    memset(bufx, 0x00, bytes);
+    memset(rX_raw, 0x00, bytes);
+    /* Then test big endian write and read */
+    TEST_EQUAL(0, mbedtls_mpi_mod_write(&rA, &m, (unsigned char *) bufx,
+                                        bytes,
+                                        MBEDTLS_MPI_MOD_EXT_REP_BE));
```

Comment:
Nitpicking, so I wont block the review. Now with bytes, they can both fit in one line:

~~~~~~
    TEST_EQUAL(0, mbedtls_mpi_mod_write(&rA, &m, (unsigned char *) bufx,
                                        bytes, MBEDTLS_MPI_MOD_EXT_REP_BE));
~~~~~~